wagtail/search/static/wagtailsearch/js/query-chooser-modal.js

```diff
@@ -0,0 +1,74 @@
+QUERY_CHOOSER_MODAL_ONLOAD_HANDLERS = {
+  chooser: function (modal, jsonData) {
+    function ajaxifyLinks(context) {
+      $('.listing a.choose-query', context).on('click', chooseQuery);
+
+      $('.pagination a', context).on('click', function () {
+        var page = this.getAttribute('data-page');
+        setPage(page);
+        return false;
+      });
+    }
+
+    var searchUrl = $('form.query-search', modal.body).attr('action');
+    var request;
+
```

Comment:
No need to commit this file, it can be deleted. It's just a left over from a different change recently to main.

Reply:
Alright. I'll delete the file entirely.

Reply:
Another question: Since the folder leading up to that point is empty, can I just delete the `static/wagtailsearch/js` path from the `search` folder as well?

Reply:
Great question. Git doesn't actually store folders. It only knows about files. You can delete or keep the empty directory and it won't make any difference.

Reply:
Wow! Thank you very much. I didn't know about that. I'm pushing again...